.github/workflows/build.yaml

```diff
@@ -5,10 +5,10 @@ on:
   push:
     branches:
       - main
-      - 'release-[0-9]+\.[0-9]+\.[0-9]+'
+      - incubating
     tags:
       - 'latest'
-      - 'v*'
+      - 'odh-v*'
```

Comment:
we need to document it somewhere so we don't miss the correct tag name when doing the releases.

Reply:
good point - I just updated the odh release process doc to reflect the new tags